### PR TITLE
Removing texts part d#161

### PR DESCRIPTION
**Pull request by amabelleS**
# Description
Removed from: TimeSelector, StopSelector, RouteSelector, OperatorSelector, MinuteSelector, LineSelector and Profile.

Next, maybe in this PR, I want to remove texts from BusToolTip. but I'm not sure where to find it in the UI, and I need to check because some translations are missing there even in the texts file...

### CODE REVIEW

File: src/pages/components/MapLayers/BusToolTip.tsx [hunk 1]
```diff
@@ -16,6 +17,7 @@ export function BusToolTip({ position, icon }: BusToolTipProps) {
   const [siriRide, setSiriRide] = useState<SiriRideWithRelatedPydanticModel | undefined>()
   const [isLoading, setIsLoading] = useState(false)
   const [showJson, setShowJson] = useState(false)
+  const { t } = useTranslation()
```
Comment (NoamGaash) [inline src/pages/components/MapLayers/BusToolTip.tsx:20]: this one can be removed

**Review by NoamGaash — APPROVED** [review]
:clap: :clap: :clap:

**Comment by NoamGaash** [comment]
the bus tooltip can be found in every map, by clicking a bus icon :bus: 
thanks for your contributions!

**Comment by shootermv** [comment]
the only problem now - is unused `t` variable at `Tooltip.tsx` file

**Comment by amabelleS** [comment]
> the only problem now - is unused `t` variable at `BusTooltip.tsx` file

That's because I want to push it to this PR and not the next one:) that is, to remove TEXTS from BusTooltip :)

**Comment by NoamGaash** [comment]
so please remove this `t` for now, and re-add it in the next PR -
the reasoning behind it is that we want to keep things "clean", without things that are in progress. it can confuse others

**Comment by amabelleS** [comment]
> so please remove this `t` for now, and re-add it in the next PR - the reasoning behind it is that we want to keep things "clean", without things that are in progress. it can confuse others

Sorry, I saw this comment just now, after I made the changes.

**Comment by NoamGaash** [comment]
@amabelleS 
Try clicking one of the busses:
image

**Review by shootermv — APPROVED** [review]

**Pull request merged**